CONTRIBUTING.md

```diff
@@ -67,6 +70,8 @@ create table "UpdatedRecords" (
 
 > Thanks Zernonia! This create table script is generated using [supabase-schema](https://github.com/zernonia/supabase-schema).
 
+More forward link is [Supabase-App](https://app.supabase.com/)
```

Comment:
Haha thanks for telling me! No wonder the other contributor also couldn't see my review comments lmao

Reply:
Lemme know I need to remove this.

Reply:
@lyqht

Reply:
@lyqht have a look, one more PR is pending 😜

Reply:
also, you can still make your PRs for the other issues you're assigned to. The code changes in this PR shouldn't conflict with the layout styling changes you make.

Reply:
@lyqht Oh I thought, If I push something on my personal repo it will reflect here. As I saw in case of layout fix 😥. Not sure about I'm newbie in these scenarios